### PR TITLE
fix: ACNA-1764 - update aio app:list:extension and aio app list:extension-points commands

### PR DESCRIPTION
**Pull request by shazron**
## Description

The extension points list cannot be hardcoded anymore since extension templates will be installed from the Template Registry.

## How Has This Been Tested?

- npm test
- aio app list extensions
- aio app list extension-points

(make sure you init an app with extensions)

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: src/commands/app/list/extension-points.js [hunk 1]
```diff
@@ -14,7 +14,6 @@ const BaseCommand = require('../../../BaseCommand')
 const aioLogger = require('@adobe/aio-lib-core-logging')('@adobe/aio-cli-plugin-app:list:extension-points', { provider: 'debug' })
 const { Flags } = require('@oclif/core')
 
-const { EXTENSION_POINT_LIST } = require('../../../lib/defaults')
```
Comment (MichaelGoberling) [inline src/commands/app/list/extension-points.js:17]: Can remove from defaults, looks to be unused elsewhere

Comment (shazron) [reply src/commands/app/list/extension-points.js:17]: good catch, will update

Comment (shazron) [reply src/commands/app/list/extension-points.js:17]: updated with new commit 0f9be2d

**Review by MichaelGoberling — APPROVED** [review]

**Comment by purplecabbage** [comment]
this should have been ACNA-1763 I think ...

**Pull request merged**